LICENSE.md

```diff
@@ -47,11 +47,10 @@ http://hg.grauw.nl/grauw-lib
 
 https://github.com/cujojs/when
 
-> Open Source Initiative OSI - The MIT License
+>  Licensed under the MIT License at:
+>  http://www.opensource.org/licenses/mit-license.php
 >
-> http://www.opensource.org/licenses/mit-license.php
->
-> Copyright (c) 2011 Brian Cavalier
+>  MIT License (c) copyright B Cavalier &amp; J Hann
```

Comment:
Do you have a link to this info?  Going by https://github.com/cujojs/when/blob/master/LICENSE.txt I'm not sure anything needed to change here.